src/main/java/de/picturesafe/search/elasticsearch/config/FieldConfiguration.java

```diff
@@ -52,6 +52,8 @@
 
     String getAnalyzer();
 
+    boolean isWithoutIndexing();
+
```

Comment:
You forgot to add it to toDocument() method.

Reply:
yes, fixed it.